DOL.WHD.Section14c.Web/src/modules/components/resultsTable/resultsTableController.js

```diff
@@ -16,10 +16,11 @@ module.exports = function(ngModule) {
     'use strict';
 
     var vm = this;
+    var tableId = $scope.tableId;
     $scope.data = [];
 
     this.initDatatable = function () {
-      let exampleId = $('#example');
+      let exampleId = $('#' + tableId);
```

Comment:
Elsewhere in the code, we are using [ES6-style template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).  For consistency's sake, this should be``$(`#${tableId}`)``